pysat/instruments/cosmic_gps.py

```diff
@@ -32,6 +33,7 @@
 - 'sonprf': 'sonPrf' files
 - 'wetprf': 'wetPrf' files
 - 'atmPrf': 'atmPrf' files
```

Comment:
Should `atmPrf` be `atmprf` for consistency?

Reply:
done...